$NetBSD: patch-Makefile,v 1.3 2026/07/08 15:44:41 micha Exp $

- Remove nonportable strip option "--strip-unneeded"
  POSIX strip doesn't support options:
  https://pubs.opengroup.org/onlinepubs/9799919799/utilities/strip.html
- Don't set root owner on install

--- Makefile.orig	2023-01-03 03:29:27.000000000 +0000
+++ Makefile
@@ -18,7 +18,10 @@ MANDIR ?= $(PREFIX)/share/man
 
 WARNINGS=-Wall -Wextra -Wshadow -Wno-unused-parameter
 OPTIMISE=-O2 -g -fno-exceptions -fno-rtti -fno-strict-aliasing -fwrapv
-STRIP_FLAGS=--strip-unneeded
+# pkgsrc: "--strip-unneeded" is not supported on all pkgsrc architectures
+#STRIP_FLAGS= --strip-unneeded
+STRIP_FLAGS=
+
 
 # default flags for compiler, preprocessor and linker
 CXXFLAGS ?= $(OPTIMISE) $(WARNINGS)
@@ -81,9 +84,9 @@ stripped: all
 
 install: stripped
 	install -d -m 755 $(DESTDIR)$(PREFIX)/bin
-	install -o root -m 755 $(PROGRAM) $(DESTDIR)$(PREFIX)/bin/
+	install -m 755 $(PROGRAM) $(DESTDIR)$(PREFIX)/bin/
 	install -d -m 755 $(DESTDIR)$(MANDIR)/man6
-	install -o root -m 644 doc/$(MAN_PAGE) $(DESTDIR)$(MANDIR)/man6/
+	install -m 644 doc/$(MAN_PAGE) $(DESTDIR)$(MANDIR)/man6/
 
 uninstall:
 	rm -f -v $(DESTDIR)$(PREFIX)/bin/$(PROGRAM)
